src/python/WMQuality/Emulators/PhEDExClient/MockPhEDExApi.py

```diff
@@ -17,16 +21,22 @@
 SITES = ['T2_XX_SiteA', 'T2_XX_SiteB', 'T2_XX_SiteC']
 _BLOCK_LOCATIONS = {}
 
-mockData = {}
+BLOCKS_PER_DATASET = 2
+FILES_PER_BLOCK = 5
+FILES_PER_DATASET = BLOCKS_PER_DATASET * FILES_PER_BLOCK
+
+MOCK_DATA = {}
 
 
 class MockPhEDExApi(object):
     """
     Version of Services/PhEDEx intended to be used with mock or unittest.mock
     """
 
-    def __init__(self, responseType="json"):
-        pass
+    def __init__(self, dict=None, responseType="json", secure=True):
+        dict = dict or {}
```

Comment:
Unfortunately, yes. This function has to have the same signature as the real PhEDEx service and since that one is sometimes called with named arguments, it even needs to be called dict. Not happy about that, but I don't really want to change the PhEDEx interface at this time.